frontend/src/components/question-creation/question-elements/TextElement.tsx

```diff
@@ -43,6 +44,7 @@ const TextElement = ({ id, data }: TextElementProps): React.ReactElement => {
       resize="none"
       minRows={1}
       as={ResizeTextarea}
+      paddingLeft="6"
```

Comment:
this differs from the design, but design asked to have all elements other than the question element be further indented